dist/index.js

```diff
@@ -163,7 +163,7 @@ exports.default = (url) => {
         href: url.href,
         path: `${url.pathname || ''}${url.search || ''}`
     };
-    if (is_1.default.string(url.port) && url.port.length !== 0) {
+    if (is_1.default.string(url.port) && url.port.length > 0) {
```

Comment:
Was there a previous PR where `dist/` wasn't regenerated? Doesn't look like these changes should come from any of the mentioned dependency PRs AFAICT.

Reply:
`dist/index.js` is the compiled action. All of the source code and dependencies are compiled into a single file and this is what the marketplace points to when a user wants to use our action.

We purposely don't commit builds until we update the action because GitHub actions allow you to specify `master`. To ensure we don't accidentally break existing users with a bad commit we gate our releases.